go/http/api.go

```diff
@@ -144,7 +144,17 @@ func (api *APIImpl) Check(w http.ResponseWriter, r *http.Request, ps httprouter.
 		remoteAddr = r.RemoteAddr
 		remoteAddr = strings.Split(remoteAddr, ":")[0]
 	}
-	checkResult := api.throttlerCheck.Check(appName, storeType, storeName, remoteAddr)
```

Comment:
Thanks for that, good naming reflecting the intent of the API method ✨ . please :shipit: